src/game/Tactical/Items.cc

```diff
@@ -336,6 +337,16 @@ static const AttachmentInfoStruct AttachmentInfo[] =
 	{XRAY_BULB,			IC_NONE,	ATTACHING_SPECIAL_ELECTRONIC_ITEM_CHECK,	-15},
 	{COPPER_WIRE,			IC_NONE,	ATTACHING_SPECIAL_ELECTRONIC_ITEM_CHECK,	+20},
 	{CERAMIC_PLATES,		IC_ARMOUR,	NO_CHECK,					0},
+
+	// extras
+	{NIGHTGOGGLES,				IC_ARMOUR,	NO_CHECK,								0},
+	{UVGOGGLES,					IC_ARMOUR,	NO_CHECK,								0},
+	{SUNGOGGLES,				IC_ARMOUR,	NO_CHECK,								0},
+
+	{ADRENALINE_BOOSTER,		IC_ARMOUR,	NO_CHECK,								0},
+	{REGEN_BOOSTER,				IC_ARMOUR,	NO_CHECK,								0},
+	{BREAK_LIGHT,				IC_ARMOUR,	NO_CHECK,								0},
+
```

Comment:
these are aligned with the line of max length, if we ever want to make the whole thing look like a table.